.azure/gpu-tests-pytorch.yml

```diff
@@ -54,6 +54,10 @@ jobs:
           image: "pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.13-cuda11.7.1"
           scope: ""
           PACKAGE_NAME: "pytorch"
+        'PyTorch | nightly':
+          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.13-cuda11.7.1"
```

Comment:
Why not create a 2.0 docker image? We'll need it later anyways

Reply:
That sounds good to me =)

Reply:
I wanted to but @Borda didn't like changes to the dockerfile

Reply:
True, I did revert it, but maybe it would be good call; sorry for moving there and back...

Reply:
as it is solution just for next a few weeks I would stick with the simple way and don't build special images as that would also mean additional adjustments on workflow side so the RC are not overwritten due to `FREEZE_REQUIREMENTS`